jwala-services/src/test/java/com/cerner/jwala/service/impl/MediaServiceImplTest.java

```diff
@@ -197,6 +204,7 @@ public void testGetMediaTypes() {
         private static final RepositoryService MOCK_MEDIA_REPOSITORY_SERVICE = mock(RepositoryService.class);
```

Comment:
I don't think these should be capitalized since they're mock instantiations of service classes, and not really enums or constants